fire/cli/info.py

```diff
@@ -9,6 +9,8 @@
 from sqlalchemy.orm import aliased
 from sqlalchemy.orm.exc import NoResultFound
 from sqlalchemy import or_
+from sqlalchemy import not_
```

Comment:
Det er nok lidt simplere at importere `not_` sammen med `or_`:

`from sqlalchemy import or_, not_`. Det er især praktisk når der er meget der importeres fra samme sted.